addons/l10n_sa_edi/models/account_journal.py

```diff
@@ -91,6 +91,10 @@ class AccountJournal(models.Model):
     l10n_sa_latest_submission_hash = fields.Char("Latest Submission Hash", copy=False,
                                                  help="Hash of the latest submitted invoice to be used as the Previous Invoice Hash (KSA-13)")
 
+    l10n_sa_branch_id = fields.Char("Branch ID", copy=False,
```

Comment:
We can not add the field like that in stable. Also, if it is not a many2one, better not to call it _id.  We can check next week for what is the best solution for this.